src/pages/yu/Login/JeonginLogin.js

```diff
@@ -1,5 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import '../Login/JeonginLogin.scss';
+
 const JeonginLogin = () => {
-  return <div>Login</div>;
+  const navigate = useNavigate();
+  const [id, setId] = useState('');
+  const [pw, setPw] = useState('');
+  const isInputValue = id.includes('@') && pw.length >= 5;
+
+  const goToMain = () => {
+    navigate('/jeongin-Main');
+  };
+
+  const saveUserId = e => {
+    setId(e.target.value);
+  };
+  const saveUserPw = e => {
+    setPw(e.target.value);
+  };
+
+  return (
+    <main className="login">
+      <div className="login_box">
+        <h1 className="westagram_login">Westagram</h1>
+        <input
+          onChange={saveUserId}
+          value={id}
+          id="LoginId"
+          className="login_id"
+          type="name"
+          placeholder=" 전화번호,
+        사용자이름 또는 이메일"
+          required
+        />
+        <input
+          onChange={saveUserPw}
+          value={pw}
+          id="LoginPw"
+          className="login_pw"
+          type="password"
+          placeholder="
+        비밀번호"
+          required
+        />
+        <button
+          id="LoginBtn"
+          className={isInputValue ? 'buttonLoginAble' : 'buttonLoginDisable'}
+          disabled={isInputValue ? false : true}
```

Comment:
```suggestion
          disabled={!isInputValue}
```
isInputValue 값 자체가 boolean 값이기에 이와 같이 구현해도 되겠습니다!

Reply:
수정 완료했습니다.! 미션 계속 진행해보도록 하겠습니다!